src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@import './variables';
+
+.product {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  box-sizing: border-box;
```

Comment:
would be better to move it to the `*` selector
```suggestion
```

Reply:
test fails if i move it from card(